circuits/circuits/send.circom

```diff
@@ -22,6 +25,25 @@ template Send(levels) {
 
     signal output new_root;
 
+    // signature verification.
+    {
+        // TODO: ensure in smart contract's force_include() fn that signature is valid, otherwise revert.
```

Comment:
I don't mind this approach, because it's nice to reduce attack surface (i.e., make sure people can't force weird invalid signatures). But there's a tradeoff with censorship resistance:

Ideally we could allow `force_include()` to have invalid signatures, and just ignore them in the circuit if they don't verify correctly. `ECRECOVER` costs 3000 gas, and that probably dominates our `force_include` function's cost. We want `force_include` to be cheap because that defines how censorship resistant we are during normal operation.

Reply:
done. in smart contract we need to ensure that slashing is not done for invalid sigs.